docs/reference/health/health.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[health-api]]
+=== Health API
+++++
+<titleabbrev>Health</titleabbrev>
+++++
+
+An experimental API that returns the health status of an {es} cluster.
+
+This API is currently experimental for internal use by Elastic software only.
+
```

Comment:
Maybe it is calling it out a third time, but I think we should include the cloud-only note too:

```suggestion

NOTE: {cloud-only}

```